src/main/java/com/oltpbenchmark/api/Procedure.java

```diff
@@ -37,6 +37,18 @@ public abstract class Procedure {
 
     private final String procName;
     private DatabaseType dbType;
+    /**
+     * LoadConfig procedure in auctionmark and seats benchmark contains CLOB on Oracle config
+     *
+     * For other databases, calling <code>SQLUtil.getString(Object)</code> after connection close is fine, but for
+     *      Oracle config, CLOBS needs to be resolved into String when the connection is open. Making this field
+     *      protected allows LoadConfig procedure to determine current database type, and decide if it should perform
+     *      the conversion.
```

Comment:
```suggestion
     * the conversion.
```

nit: extra whitespace on these comment lines

Reply:
But also, we probably don't need that comment here at all.  Only in the places where it's actually relevant (auctionmark and seats load procedures).